src/one_dragon/base/controller/pc_controller_base.py

```
@@ -190,8 +190,10 @@ def close_game(self):
         try:
             self.game_win.get_win().close()
             log.info('关闭游戏成功')
+            return 0
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)